corehq/apps/fixtures/tests/test_views.py

```diff
@@ -282,3 +284,39 @@ def can_view_some_reports(domain):
                 "description": "A Table",
             }],
         )
+
+
+class UpdateTablesTests(TestCase):
+    def test_cannot_update_synced_fixture(self):
+        request = self._create_request('GET')
+        fixture = LookupTable.objects.create(domain='test-domain', tag='test-fixture', is_synced=True)
+
+        # ensure does not raise an exception
+        update_tables(request, 'test-domain', fixture.id)
```

Comment:
This test doesn't seem to verify that the lookup table was not updated, only that `update_tables()` does not raise an exception. Should it check for that, or maybe renamed to something like `test_update_synced_fixture_does_not_raise`?

Reply:
Removed this test. Requirements had changed, and I didn't realize this was never updated.